src/pages/graphql/payment-services-extension/workflows/index.md

```diff
@@ -16,4 +16,4 @@ If you have installed [Payment Services for Adobe Commerce](https://commercemark
 
 * [Minicart](../workflows/minicart.md)---This workflow describes the additional flow of requests and responses required to place an order after adding a product to your cart with the Payment Services extension enabled.
 
-* [Vault](../workflows/vault.md)---This workflow shows the workflow for storing a payment method during checkout.
+* [Checkout using a vaulted card](../workflows/vaulted-card.md)---This workflow describes a checkout authorization using a vaulted payment method.
```

Comment:
Vaulted card or vaulted payment method? Let's use consistent naming. As we use "Vaulted card" in most other changes in this PR, I propose changing it to "Vaulted card".

```suggestion
* [Checkout using a vaulted card](../workflows/vaulted-card.md)---This workflow describes a checkout authorization using a vaulted card.
```